### DISCUSSION TITLE
Python errors / downloading datasets

### DISCUSSION THREAD

**Discussion by ClarkKentIsSuperman** [Giskard-AI/giskard-oss#2003] [General]
How are the local datasets supposed to be downloaded?  I was following some of the intro demos, and got an error saying the prompt CSV wasn't found:

```
FileNotFoundError: [Errno 2] No such file or directory: '/Users/XXXXX/py311_env/lib/python3.11/site-packages/giskard/llm/loaders/local/prompt_injections.csv'
```
and I saw the "local" folder is not there, but just a prompt downloader.py is there, but seems like I can't call the .py directly (maybe its mean to be run as a package/module):

```
 File "/Users/XXXXXX/py311_env/lib/python3.11/site-packages/giskard/llm/loaders/prompt_injections.py", line 8, in <module>
    from ...datasets.base import Dataset
ImportError: attempted relative import with no known parent package
```

Any suggestions on what I missed?

**Comment by kevinmessiaen** [comment]
Hey @ClarkKentIsSuperman 

It seems that those file aren't packaged properly when installing the wheel, I'll take a look to fix that

**Reply by ClarkKentIsSuperman** [reply]
thanks, for now I just manually downloaded and created the folder.

**Reply by kevinmessiaen** [reply]
No problem, it's fixed now on main and should be included in the next release
